docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx

```diff
@@ -161,34 +161,34 @@ To add a Task to the end of a Mission:
     replaced accordingly depending on the task selected (waypoint icons will keep the colours
     assigned to them based on placement).
 
-    -   **Dock UGV:** <img src="/img/outdoornav_images/dockTask_40px.png" width="30px"/>
+    -   <img src="/img/outdoornav_images/dockTask_40px.png" width="30px"/> <strong>Dock UGV: </strong>
         Will dock the UGV to begin charging the UGVs

```

Comment:
UGV's

Reply:
I think in this case it should be singular right? This task is for docking the UGV but I could be wrong on whether it should be plural or not.

Reply:
oh shoot, apologies  I was looking at the wrong UGV